src/Base/IObjectDependency.cs

```diff
@@ -11,13 +11,13 @@
     public interface IObjectDependency
```

Comment:
@Karnah 
I understand you intention to make the name of the Method more easy to understand.
I have no issue to change the comment, but please let the Method names, even if the
name is not perfect. This is a public Interface an even if nearly nobody will use it I don't
like to change it, for this small reason.

Reply:
Sorry, I could not understand what you mean. Should I return names OnReferencesRemoved and OnAllReferencesRemoved? Or use something else?